packages/effect/src/ManagedRuntime.ts

```diff
@@ -29,7 +52,8 @@ export declare namespace ManagedRuntime {
  * @since 2.0.0
  * @category models
  */
-export interface ManagedRuntime<in R, out ER> extends Pipeable {
+export interface ManagedRuntime<in R, out ER> extends Pipeable, Effect.Effect<Runtime.Runtime<R>, ER> {
```

Comment:
You can remove Pipeable as Effect is already Pipeable.

Reply:
oh